libs/ui-lib/lib/ocm/components/clusterDetail/utils.tsx

```diff
@@ -98,3 +97,21 @@ export const getMostSevereHostStatus = (hosts: Host[]) => {
   }
   return status;
 };
+
+export const download = (fileUrl?: string, dataBlob?: Blob, fileName?: string) => {
```

Comment:
Why are you exporting this function?
Is it used anywhere else?

Reply:
True, we don't need to export it.